scripts/check_requirements.py

```diff
@@ -1,18 +1,29 @@
 #! /usr/bin/env python3
 import os
+import sys
 import pkg_resources
 from pkg_resources import DistributionNotFound, VersionConflict
 
+args = sys.argv
+
+# verbose flag
+if '-v' in args:
```

Comment:
`verbose = True if '-v' in args else False`

is always a little nicer to read for me.

Reply:
change it, but only for you ;-)